.ci_support/build_all.py

```diff
@@ -207,6 +242,50 @@ def build_folders(recipes_dir, folders, arch, channel_urls):
         conda_build.api.build([recipe], config=get_config(arch, channel_urls))
 
 
+def build_folders_rattler_build(
+    recipes_dir: str, platform, arch, channel_urls: list[str]
+):
+    config = get_config(arch, channel_urls)
+
+    # Remove the example recipes to ensure that they are not also build.
+    for example_recipe in EXAMPLE_RECIPE_FOLDERS:
+        rmtree(os.path.join(recipes_dir, example_recipe), ignore_errors=True)
+
+    # Determine the locations for the variant config files.
+    specs = OrderedDict()
```

Comment:
This is less a question for this PR and more a topic overall across our infrastructure, but at least I thought I'd note it here: https://github.com/conda-forge/conda-smithy/issues/2033

Reply:
```suggestion
    specs = {}
```